test/e2e/specs/wp-stats-insights-spec.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import LoginFlow from '../lib/flows/login-flow.js';
+
+import NavBarComponent from '../lib/components/nav-bar-component.js';
+
+import StatsPage from '../lib/pages/stats-page.js';
+
+import * as driverManager from '../lib/driver-manager.js';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const screenSize = driverManager.currentScreenSize();
+
+let driver;
+
+before( async function() {
+	this.timeout( startBrowserTimeoutMS );
+	driver = await driverManager.startBrowser();
+} );
+
+describe( 'Stats: (' + screenSize + ') @parallel', function() {
+	this.timeout( mochaTimeOut );
+	describe( 'Log in as user', function() {
+		step( 'Can log in as user', async function() {
+			this.loginFlow = new LoginFlow( driver );
+			return await this.loginFlow.login( { useFreshLogin: true } );
+		} );
+
+		step( 'Can open the sidebar', async function() {
+			const navBarComponent = await NavBarComponent.Expect( driver );
+			await navBarComponent.clickMySites();
+		} );
+
+		describe( 'Can navigate to the stats insights page', function() {
+			let statsPage;
+
+			step( 'Can open the stats page', async function() {
+				statsPage = await StatsPage.Expect( driver );
```

Comment:
Since Stats page is not displayed on mobile by default, as you can see in the [screenshot](https://45030-102822243-gh.circle-artifacts.com/0/home/circleci/wp-calypso/wp-calypso/test/e2e/screenshots/FAILED-EN-MOBILE-can-open-the-stats-insights-page-1580210143666.png) of the failed test, additional click on `Stats` is needed. Before this line try adding this:

```
const sidebarComponent = await SidebarComponent.Expect( driver );
await sidebarComponent.selectStats();
```

Reply:
Done, thank you! 🙂